eviloauth/routes.py

```diff
@@ -1,5 +1,9 @@
 import logging
 import requests
+from . import app, cache
```

Comment:
Change back to from eviloauth import app, cache. Don't use relative imports, always:

from eviloauth.module import stuff. This goes for the following lines.